lib/ai_bot/tools/web_browser.rb

```diff
@@ -0,0 +1,114 @@
+# frozen_string_literal: true
+
+module DiscourseAi
+  module AiBot
+    module Tools
+      class WebBrowser < Tool
+        def self.signature
+          {
+            name: name,
+            description:
+              "Visits a web page, retrieves the HTML content, extracts the main content, converts it to plain text, and returns the result.",
```

Comment:
I can't remember, are these texts only sent to the LLM? If not and they are shown to the user then we should I18n them.

Reply:
for llm only :)